Formula/ohdear-cli.rb

```diff
@@ -0,0 +1,17 @@
+class OhdearCli < Formula
+  desc "Tool to manage your Oh Dear sites"
+  homepage "https://github.com/nunomaduro/ohdear-cli"
```

Comment:
ohdearapp/ohdear-cli

Reply:
Argh, thank you! Shows how old my original Brew formula is.

Reply:
this is nice!